--- origsrc/strigi-0.6.3/ConfigureChecks.cmake	2009-01-12 15:50:22.000000000 -0600
+++ src/strigi-0.6.3/ConfigureChecks.cmake	2009-01-15 01:44:06.687125000 -0600
@@ -7,7 +7,9 @@ CHECK_INCLUDE_FILE_CXX(ndir.h HAVE_NDIR_
 CHECK_INCLUDE_FILE_CXX(stddef.h HAVE_STDDEF_H)          # unused !
 CHECK_INCLUDE_FILE_CXX(sys/dir.h HAVE_SYS_DIR_H)        # src/streams/strigi/stgdirent.cpp/.h
 CHECK_INCLUDE_FILE_CXX(sys/ndir.h HAVE_SYS_NDIR_H)      # src/streams/strigi/stgdirent.cpp/.h
+IF(WIN32)
 CHECK_INCLUDE_FILE_CXX(windows.h HAVE_WINDOWS_H)        # src/streamindexer/filelister.cpp
+ENDIF(WIN32)
 
 # files that may define the u?int{8,16,32,54}_t types
 CHECK_INCLUDE_FILE_CXX(socket.h HAVE_SOCKET_H)
