--- origsrc/strigi-0.6.3/src/daemon/strigithread.cpp	2009-01-12 15:50:14.000000000 -0600
+++ src/strigi-0.6.3/src/daemon/strigithread.cpp	2009-01-15 11:26:01.619688900 -0600
@@ -29,7 +29,9 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/resource.h>
+#ifndef __CYGWIN__
 #include <sys/syscall.h>
+#endif
 
 // define two enums and a constant for use of ioprio
 enum {
--- origsrc/strigi-0.6.3/src/streamanalyzer/indexpluginloader.cpp	2009-01-12 15:50:13.000000000 -0600
+++ src/strigi-0.6.3/src/streamanalyzer/indexpluginloader.cpp	2009-01-16 00:49:47.369250000 -0600
@@ -44,6 +44,10 @@ typedef void* StgModuleType;
 typedef HMODULE StgModuleType;
 #endif
 
+#ifndef RTLD_LOCAL
+#define RTLD_LOCAL 0
+#endif
+
 vector<string> getdirs(const string& direnv) {
     vector<string> dirs;
     string::size_type lastp = 0;
