--- origsrc/strigi-0.6.3/src/luceneindexer/cluceneindexreader.h	2009-01-12 15:50:09.000000000 -0600
+++ src/strigi-0.6.3/src/luceneindexer/cluceneindexreader.h	2009-01-15 01:42:16.952750000 -0600
@@ -22,6 +22,7 @@
 
 #include <strigi/strigiconfig.h>
 #include "indexreader.h"
+#include "tcharutils.h"
 
 #include <map>
 #include <time.h>
--- origsrc/strigi-0.6.3/src/luceneindexer/tcharutils.h	2009-01-12 15:50:09.000000000 -0600
+++ src/strigi-0.6.3/src/luceneindexer/tcharutils.h	2009-01-15 02:26:52.296500000 -0600
@@ -25,6 +25,12 @@
 #include <string>
 #include <strigi/strigiconfig.h>
 
+#ifndef __GLIBCXX_USE_WCHAR_T
+namespace std {
+typedef basic_string<wchar_t> wstring;
+}
+#endif
+
 std::string wchartoutf8(const wchar_t*);
 std::wstring utf8toucs2(const char*);
 std::string wchartoutf8(const std::wstring&);
