--- origsrc/strigi-0.6.3/src/streamanalyzer/analyzerloader.cpp	2009-01-12 15:50:13.000000000 -0600
+++ src/strigi-0.6.3/src/streamanalyzer/analyzerloader.cpp	2009-01-16 00:56:20.259875000 -0600
@@ -111,12 +111,12 @@ AnalyzerLoader::loadPlugins(const char* 
         if ((strncmp(ent->d_name + iOffset, "strigita_", 9) == 0
                 || strncmp(ent->d_name + iOffset, "strigiea_", 9) == 0
                 || strncmp(ent->d_name + iOffset, "strigila_", 9) == 0)
-#ifdef WIN32
+#if defined(WIN32) || defined(__CYGWIN__)
                 && strcmp(ent->d_name+len-4, ".dll") == 0
 # if defined(_MSC_VER)
                 && strncmp(ent->d_name, "msvc_", 5) == 0) {
 #elif defined(__CYGWIN__)
-                && strncmp(ent->d_name, "cyg_", 4) == 0) {
+                ) {
 # else
                 && strncmp(ent->d_name, "mingw_", 6) == 0) {
 # endif
--- origsrc/strigi-0.6.3/src/streamanalyzer/indexpluginloader.cpp	2009-01-12 15:50:13.000000000 -0600
+++ src/strigi-0.6.3/src/streamanalyzer/indexpluginloader.cpp	2009-01-16 00:49:47.369250000 -0600
@@ -182,7 +182,7 @@ IndexPluginLoader::loadPlugins(const cha
     }
     struct dirent* ent = readdir(dir);
     string prefix("strigiindex_");
-#ifdef WIN32
+#if defined(WIN32) || defined(__CYGWIN__)
     string suffix(".dll");
 #else
     string suffix(".so");
