--- origsrc/strigi-0.7.5/libstreamanalyzer/CMakeLists.txt	2011-06-02 04:29:26.000000000 -0500
+++ src/strigi-0.7.5/libstreamanalyzer/CMakeLists.txt	2011-06-12 04:08:10.877517300 -0500
@@ -50,9 +50,9 @@ set (LIB_DESTINATION "${CMAKE_INSTALL_PR
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH  TRUE)
 set(CMAKE_INSTALL_RPATH                ${LIB_DESTINATION} )
 
-if(NOT WIN32 AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
+if(NOT WIN32 AND NOT CYGWIN AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
 	add_definitions(-fPIC)
-endif(NOT WIN32 AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
+endif(NOT WIN32 AND NOT CYGWIN AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
 
 ##### environment inspection #####
 
--- origsrc/strigi-0.7.5/libstreams/CMakeLists.txt	2011-06-02 04:28:57.000000000 -0500
+++ src/strigi-0.7.5/libstreams/CMakeLists.txt	2011-06-12 04:07:51.584283900 -0500
@@ -36,9 +36,9 @@ file(RELATIVE_PATH INCLUDE_DESTINATION $
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH  TRUE)
 set(CMAKE_INSTALL_RPATH                ${LIB_DESTINATION} )
 
-if(NOT MSVC AND NOT MINGW AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
+if(NOT MSVC AND NOT MINGW AND NOT CYGWIN AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
 	add_definitions(-fPIC)
-endif(NOT MSVC AND NOT MINGW AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
+endif(NOT MSVC AND NOT MINGW AND NOT CYGWIN AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
 
 ##### environment inspection #####
 
--- origsrc/strigi-0.7.5/strigidaemon/CMakeLists.txt	2011-04-23 06:55:09.000000000 -0500
+++ src/strigi-0.7.5/strigidaemon/CMakeLists.txt	2011-06-12 04:07:35.999856700 -0500
@@ -33,9 +33,9 @@ endif(CMAKE_SYSTEM MATCHES "Linux")
 set (LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)" )
 set (LIB_DESTINATION "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE STRING "Library directory name" FORCE)
 
-if(NOT MSVC AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
+if(NOT MSVC AND NOT CYGWIN AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
 	add_definitions(-fPIC)
-endif(NOT MSVC AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
+endif(NOT MSVC AND NOT CYGWIN AND NOT CMAKE_SYSTEM MATCHES "SunOS-5*.")
 
 ##### environment inspection #####
 

