--- origsrc/strigi-0.7.2/src/daemon/CMakeLists.txt	2010-04-30 03:20:21.619957800 -0500
+++ src/strigi-0.7.2/src/daemon/CMakeLists.txt	2010-04-30 03:21:32.978039300 -0500
@@ -102,6 +102,7 @@ ENDIF(CMAKE_LIB_SOCKET_HAS_CONNECT)
 
 install(TARGETS strigidaemon searchclient #strigiconfupdater
 	RUNTIME DESTINATION bin
+	ARCHIVE DESTINATION ${LIB_DESTINATION}
 	LIBRARY DESTINATION ${LIB_DESTINATION})
 install(FILES asyncsocketclient.h asyncsocket.h clientinterface.h
 	DESTINATION include/strigi)
--- origsrc/strigi-0.7.2/src/htmlgui/CMakeLists.txt	2010-02-03 13:03:07.000000000 -0600
+++ src/strigi-0.7.2/src/htmlgui/CMakeLists.txt	2010-04-30 03:28:12.972917700 -0500
@@ -10,6 +10,9 @@ set_target_properties(strigihtmlgui PROP
 
 target_link_libraries(strigihtmlgui searchclient streamanalyzer)
 
-install(TARGETS strigihtmlgui DESTINATION ${LIB_DESTINATION})
+install(TARGETS strigihtmlgui LIBRARY DESTINATION ${LIB_DESTINATION}
+                              RUNTIME DESTINATION bin
+                              ARCHIVE DESTINATION ${LIB_DESTINATION})
+
 install(FILES strigihtmlgui.h
 	DESTINATION include/strigi)
